pkg/solana/config/multinode.go

```diff
@@ -1,87 +1,163 @@
 }
 
 func (c *MultiNode) PollFailureThreshold() uint32 {
-	return c.pollFailureThreshold
+	return *c.pollFailureThreshold
 }
 
 func (c *MultiNode) PollInterval() time.Duration {
-	return c.pollInterval
+	return c.pollInterval.Duration()
 }
 
 func (c *MultiNode) SelectionMode() string {
-	return c.selectionMode
+	return *c.selectionMode
 }
 
 func (c *MultiNode) SyncThreshold() uint32 {
-	return c.syncThreshold
+	return *c.syncThreshold
 }
 
 func (c *MultiNode) NodeIsSyncingEnabled() bool {
-	return c.nodeIsSyncingEnabled
+	return *c.nodeIsSyncingEnabled
 }
 
+func (c *MultiNode) LeaseDuration() time.Duration { return c.leaseDuration.Duration() }
+
 func (c *MultiNode) FinalizedBlockPollInterval() time.Duration {
-	return c.finalizedBlockPollInterval
+	return c.finalizedBlockPollInterval.Duration()
 }
 
-func (c *MultiNode) EnforceRepeatableRead() bool {
-	return c.enforceRepeatableRead
-}
+func (c *MultiNode) EnforceRepeatableRead() bool { return *c.enforceRepeatableRead }
 
-func (c *MultiNode) DeathDeclarationDelay() time.Duration {
-	return c.deathDeclarationDelay
-}
+func (c *MultiNode) DeathDeclarationDelay() time.Duration { return c.deathDeclarationDelay.Duration() }
 
 func (c *MultiNode) NodeNoNewHeadsThreshold() time.Duration {
-	return c.nodeNoNewHeadsThreshold
+	return c.nodeNoNewHeadsThreshold.Duration()
 }
 
 func (c *MultiNode) NoNewFinalizedHeadsThreshold() time.Duration {
-	return c.noNewFinalizedHeadsThreshold
+	return c.noNewFinalizedHeadsThreshold.Duration()
 }
 
-func (c *MultiNode) FinalityDepth() uint32 {
-	return c.finalityDepth
-}
+func (c *MultiNode) FinalityDepth() uint32 { return *c.finalityDepth }
 
-func (c *MultiNode) FinalityTagEnabled() bool {
-	return c.finalityTagEnabled
-}
+func (c *MultiNode) FinalityTagEnabled() bool { return *c.finalityTagEnabled }
 
-func (c *MultiNode) FinalizedBlockOffset() uint32 {
-	return c.finalizedBlockOffset
-}
+func (c *MultiNode) FinalizedBlockOffset() uint32 { return *c.finalizedBlockOffset }
 
 func (c *MultiNode) SetDefaults() {
-	// TODO: Set defaults for MultiNode config https://smartcontract-it.atlassian.net/browse/BCI-4065
-	c.multiNodeEnabled = false
+	c.multiNodeEnabled = ptr(false)
+
+	// Node Configs
+	c.pollFailureThreshold = ptr(uint32(5))
+	c.pollInterval = config.MustNewDuration(10 * time.Second)
+
+	c.selectionMode = ptr(client.NodeSelectionModePriorityLevel)
+
+	c.syncThreshold = ptr(uint32(5))
+
+	// Period at which we verify if active node is still highest block number
+	c.leaseDuration = config.MustNewDuration(time.Minute)
+
+	c.nodeIsSyncingEnabled = ptr(false)
+	c.finalizedBlockPollInterval = config.MustNewDuration(5 * time.Second)
+	c.enforceRepeatableRead = ptr(true)
+	c.deathDeclarationDelay = config.MustNewDuration(10 * time.Second)
+
+	// Chain Configs
+	c.nodeNoNewHeadsThreshold = config.MustNewDuration(10 * time.Second)
+	c.noNewFinalizedHeadsThreshold = config.MustNewDuration(10 * time.Second)
+	c.finalityDepth = ptr(uint32(0))
```

Comment:
I'm not sure the helper function is adding much here. Wouldn't this be cleaner?

```
c.finalityDepth = &uint32(0)
c.finalityTagEnabled = &true
c.finalizedBlockOffset = &uint32(0)
```

Reply:
That syntax actually doesn't work in Go! We would have to do the following instead:

```
c.finalityDepth = new(uint32)
c.finalityDepth = 0

c.finalityTagEnabled = new(bool)
*c.finalityTagEnabled = true
...
```

This ends up adding a lot of lines. In the case where our default value is not 0 or false we would have to add a second line to set the value after the `new` pointer initialization. If you think this approach is still better then I don't mind changing it.

Reply:
oh! If it doesn't work in Go, then I'd just keep the helper function. Weird that the syntax doesn't work though!